installer/mac/postinstall.sh

```diff
@@ -47,9 +47,9 @@ rm -f $tempFilePath
 ## For launching symphony with sandbox enabled, create a shell script that is used as the launch point for the app
 EXEC_PATH=$installPath/Symphony.app/Contents/MacOS
 exec $EXEC_PATH/Symphony --install $newPath
-mv $EXEC_PATH/Symphony $EXEC_PATH/Symphony-bin
-cat > $EXEC_PATH/Symphony << EOT
-#!/bin/sh
-exec "\${0%/*}/Symphony-bin" --enable-sandbox \$@
-EOT
-chmod 755 $EXEC_PATH/Symphony
\ No newline at end of file
+chmod 755 $EXEC_PATH/Symphony
+# mv $EXEC_PATH/Symphony $EXEC_PATH/Symphony-bin
+# cat > $EXEC_PATH/Symphony << EOT
+# #!/bin/sh
+# exec "\${0%/*}/Symphony-bin" --enable-sandbox \$@
```

Comment:
this shouldn't be needed anymore if removing sandbox here or are you just keeping around for later?